parlai/crowdsourcing/tasks/model_chat/model_chat_blueprint.py

```diff
@@ -445,10 +446,14 @@ def __init__(
         )
 
     def _get_shared_models(self, args: "DictConfig") -> Dict[str, dict]:
-        _ = args  # Not needed
-        models_needed = list(self.conversations_needed.keys())
-        active_models = [m for m in models_needed if self.conversations_needed[m] > 0]
-        return TurkLikeAgent.get_bot_agents(args=args, active_models=active_models)
+        with open(args.blueprint.model_opt_path) as f:
+            all_model_opts = yaml.load(f.read())
+        active_model_opts = {
+            model: opt
+            for model, opt in all_model_opts.items()
+            if self.conversations_needed[model] > 0
```

Comment:
oh great i just suggested this in the form

Reply:
Yeah - your suggestion still applies to image chat, though, for which `self.conversations_needed` isn't used